maintainers/maintainer-list.nix

```diff
@@ -1743,6 +1743,12 @@
     githubId = 113123021;
     name = "Ashvith Shetty";
   };
+  asininemonkey = {
+    email = "nixpkgs@asininemonkey.com";
+    github = "asininemonkey";
+    githubId = 65740649;
+    name = "Jose Cardoso";
+  };
```

Comment:
this should be its own commit called: `maintainers: add asininemonkey`

Reply:
I've broken this out as requested.